test/integration/cuda/test_cuda_analytical_rosenbrock.cpp

```diff
@@ -18,6 +18,7 @@ TEST(AnalyticalExamplesCudaRosenbrock, Troe)
       micm::RosenbrockSolverParameters::ThreeStageRosenbrockParameters());
```

Comment:
Looks like all the CUDA tests fail to compile. Also, do we want to test all the rosenbrock parameters for the CUDA tests as well?